tempesta_fw/http_parser.c

```diff
@@ -877,6 +895,12 @@ __FSM_STATE(RGen_OWS) {							\
 
 /**
  * Parse Connection header value, RFC 2616 14.10.
+ *
+ * Most likely that Connection header contains only "close" or "keep-alive"
+ * tokens. If Connection header contains other tokens, message flag
+ * TFW_HTTP_CONN_EXTRA is set to indicate that hop-by-hop headers are set
+ * and must be processed later. Each value will be placed to its chunk marked
+ * with TFW_STR_VALUE flag.
```

Comment:
Each chunk of a token other than those mentioned above is marked with `TFW_STR_VALUE` flag.